pbr/pbr.py

```diff
@@ -144,18 +147,20 @@ def main():
             camera_loc, hdr_data["mask_path"], env_info, len(robots) + 1
         )
         print(points_on_field)
+        world_points = util.generate_moves(field.obj)
```

Comment:
Should probably set the z and radius?

Reply:
They are registered as default args so that the z is always on the field and the radius is configurable